rosidl_generator_cpp/resource/msg__traits.hpp.template

```diff
@@ -0,0 +1,79 @@
+// generated from rosidl_generator_cpp/resource/msg__traits.hpp.template
+
+@#######################################################################
+@# EmPy template for generating <msg>__traits.hpp files
+@#
+@# Context:
+@#  - spec (rosidl_parser.MessageSpecification)
+@#    Parsed specification of the .msg file
+@#  - subfolder (string)
+@#    The subfolder / subnamespace of the message
+@#    Either 'msg' or 'srv'
+@#  - get_header_filename_from_msg_name (function)
+@#######################################################################
+@
+@{
+from rosidl_generator_cpp import MSG_TYPE_TO_CPP
+
+cpp_namespace = '%s::%s::' % (spec.base_type.pkg_name, subfolder)
+
+cpp_fixed_types = ['bool', 'char', 'uint8_t', 'int8_t', 'uint16_t', 'int16_t', 'uint32_t', 'int32_t', 'uint64_t', 'int64_t', 'float', 'double']
+}@
+#include <stdint.h>
+#include <array>
+#include <string>
+#include <vector>
+
+#ifndef __ALL_MSG__BASIC_TEMPLATES
```

Comment:
This should use a name more related to this generator. E.g. `__ROSIDL_GENERATOR_CPP_TRAIT_TEMPLATES`